core/drivers/crypto/crypto_api/include/drvcrypt_cipher.h

```diff
@@ -0,0 +1,73 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2018-2020 NXP
+ *
+ * Cipher interface calling the HW crypto driver.
+ */
+#ifndef __DRVCRYPT_CIPHER_H__
+#define __DRVCRYPT_CIPHER_H__
```

Comment:
I think it is preferred to not use `__` has prefix for macro, see https://github.com/OP-TEE/optee_client/pull/138#discussion_r368841263.

Reply:
I agree with you.
However `drvcrypt_math/hash/acipher.h` already have been merged with `__DRVCRYPT_*_H__` macros ...
I prefer to merge it as is and fix all header files in the scope of another PR.

Reply:
Please keep those `__`, OP-TEE core is the system.

Reply:
thanks @jenswi-linaro for clarification.